etc/nginx/conf/nginx.conf.j2

```diff
@@ -118,6 +123,7 @@ http {
             {% if location.issuer_dn_regex %}
             if ($http_ssl_client_i_dn ~* ^(?!({{ location.issuer_dn_regex }})$)(\w*)) {
                 # Enable logging to show ssl_client_i_dn
+                set $issuer_dn '{{ location.issuer_dn }}';
```

Comment:
How will this work with multiple locations? The log format is set on a root level.

Reply:
It should work with multiple locations because the scope of the variable is the HTTP request. From the Nginx documantation: `Requests do not interfere with each other even if they are referencing a variable with the same name.`